tests/test_immutableset.py

```diff
@@ -224,3 +225,34 @@ def type_annotations() -> int:
         source = (1, 2, 3)
         dict1 = immutableset(source)
         return next(iter(dict1))
+
+    # A reminder that when patching, it must be done where used, not where it is defined.
+    @patch("immutablecollections._immutableset.DICT_ITERATION_IS_DETERMINISTIC", False)
+    def test_dict_iteration_is_not_deterministic(self):
+        source = {"b": 2, "c": 3, "a": 1}
+        value_error_regex = "Attempting to initialize an ImmutableSet from a dict view."
```

Comment:
Very minor, but note that since this is a regex, the `.` is not interpreted literally. This doesn't cause any trouble, but I would escape it just for good practice here and elsewhere.